lib/url.js

```diff
@@ -28,6 +28,8 @@ const { hexTable } = require('internal/querystring');
 
 const errors = require('internal/errors');
 
+var { spliceOne } = require('internal/util');
```

Comment:
As this module already uses `const`, maybe we can use `const` here too?

Reply:
fixed